src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -713,6 +729,27 @@ public BitbucketTeam getTeam() throws IOException, InterruptedException {
         return getRepositories(null);
     }
 
+    /**
+     * Returns the account of the current owner or {@code null} if the current owner is not an account user.
+     *
+     * @return the account profile of the current owner, or {@code null} if current owner is not an account user.
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    @CheckForNull
+    private BitbucketCloudAccount getUser() throws IOException, InterruptedException {
+        String url = UriTemplate.fromTemplate(V2_API_USERS + "{/owner}")
```

Comment:
@car-roll  /users/{username} is going away. This should simply be  a call to /user API.